src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/MixedMode_x64_VS2013_Default.dll.sarif

```diff
@@ -1017,7 +1017,7 @@
                   "text": "'{0}' was not evaluated for check '{1}' as the analysis is not relevant based on observed metadata: {2}."
                 }
               },
-              "helpUri": "https://github.com/microsoft/binskim/blob/main/docs/BinSkimRules.md#rule-BA6004EnableCOMDATFolding"
+              "helpUri": "https://github.com/microsoft/binskim/blob/main/docs/BinSkimRules.md#rule-BA6004EnableComdatFolding"
```

Comment:
>BA6004EnableComdatFolding [](http://example.com/codeflow?start=99&length=25)

Hm. I changed casing of a local file and this appears to be the result. This very likely will cause edits on other dev client boxes. So I plan to delete and recreate this file so the casing is forced into the change.